tests/integration/refextract/test_refextract_tasks.py

```diff
@@ -39,7 +39,11 @@ def test_create_journal_kb_file(app):
         create_journal_kb_file()
 
     with open(path, 'r') as fd:
-        assert 'JOURNAL OF HIGH ENERGY PHYSICS---JHEP' in fd.read()
+        journal_kb = fd.read()
+
+        assert 'JHEP---JHEP' in journal_kb
+        assert 'THE JOURNAL OF HIGH ENERGY PHYSICS JHEP---JHEP' in journal_kb
+        assert 'JOURNAL OF HIGH ENERGY PHYSICS---JHEP' in journal_kb
```

Comment:
maybe add a test that exercises normalization?

Reply:
Actually this is what happens in `THE JOURNAL OF HIGH ENERGY PHYSICS JHEP---JHEP` as it would be `THE JOURNAL OF HIGH ENERGY PHYSICS (JHEP)---JHEP` since this is what's in http://inspire-nightly.cern.ch/api/journals/1213103:
```JSON
{
    "journal_title": {
        "title": "The Journal of High Energy Physics (JHEP)"
    }
}
```

I can make this clearer, though.